src/measure/OSRunner.hpp

```diff
@@ -231,6 +231,17 @@ namespace measure {
                                                                                        const std::map<std::string, OSArgument>& user_arguments,
                                                                                        const openstudio::Workspace& workspace);
 
+    /** Call this method to retrieve the values of OSArguments of all types. */
+    Json::Value getArgumentValues(std::vector<OSArgument>& script_arguments, const std::map<std::string, OSArgument>& user_arguments);
```

Comment:
We do we pass both script_arguments and user_arguments?

Reply:
It's becasue the user_arguments could include arguments are AREN'T actual measure arguments?

Reply:
Yes, `script_arguments` gives us the argument name, and `user_arguments` is passed into all `getXXXArgumentValue` methods.